sa/model_test.go

```diff
@@ -35,3 +39,63 @@ func TestModelToRegistrationNonNilContact(t *testing.T) {
 		t.Errorf("Expected empty Contact field, got %#v", reg.Contact)
 	}
 }
+
+func TestV2AuthzModel(t *testing.T) {
```

Comment:
I think the logic in `modelToAuthzPB` and `authzPBToModel` are complicated enough that there should be more test cases in this unit test. At the least I think the `deactivated` case I mentioned above would be good to capture here. Maybe also a test case for a non-v2 authz being passed to `authzPBToModel`.